content/learn/book/welcome/hello-bevy/_index.md

```diff
@@ -0,0 +1,144 @@
++++
+title = "Hello, Bevy!"
+weight = 3
+template = "book-section.html"
+page_template = "book-section.html"
++++
+
+So, you've just made a project with Bevy. Great! However, currently, it isn't doing things 'the Bevy way'. Let's fix.
+
+## Bevy App Structure
+
+Bevy programs, which include games, tools and more, are conventially named 'Apps' and currently our Bevy program doesn't really follow the app structure or how bevy apps work.
+
+Below is a basic outline of how your Bevy Apps should look.
+
+```rust
+// MODULES (only the first line is required)
+use bevy::prelude::*;
+
+use another::module::*;
+
+mod some_file;
+mod another_file;
+
+// COMPONENTS AND RESOURCES
+
+struct Player;
+
+struct Health {
+    value: u32,
+}
+
+impl Health {
+    pub fn start() -> Self {
+       Self {
+           value: 100,
+       } 
+    }
+}
+// SYSTEMS
+
+fn setup(mut commands: Commands) {
+    commands.spawn()
+        .insert(Player)
+        .insert(Health::start());
+    println!("Setup Complete")
+}
+
+// MAIN
+
+fn main() {
+    App::build()
+        .add_system(setup.system())
+        .run();
+}
+```
+
+Bear in mind, the order of everything beyond modules is completely up to you.
+
+In detail:
+- First, you should import your modules. In order for a bevy program to work properly, it is **ESSENTIAL** that you import the bevy prelude, as this contains the App struct and other things required for bevy to work.
+
+- Secondly, you should define your components and implementations. As displayed, a component can **optionally have values and implementations** which can be used when creating an entity
+
+- Thirdly, you should define your systems which are the logic of the game. In this example, we have a `setup()` function which spawns a new entity and inserts 2 componenets, Player and Health. We can use implementations in functions in the insertion. See how it links together!
+
+- Finally, we have the main function which defines a Bevy App, adds any systems, builds and runs our new App. Whenever, you define a new system, you need to add it to this decleration. This can get messy quick, which is why you usually seperate these things into **plugins** (talked about in a later chapter).
+
+A lot of this, you don't need to wory about right now. Just remember that:
+- You add systems to your app in the main function and
+- You define systems that can act on components
```

Comment:
```suggestion
- You define systems that can act on the component data of entities
```

The interplay with entities is important here: you're not just operating on columns of data at once, and talking about how entities have components ties the mental model together a bit better.